frontend/packages/kubevirt-plugin/integration-tests-cypress/tests/tier1-b/win-guest-tools-mount.spec.ts

```diff
@@ -0,0 +1,96 @@
+import { OperatingSystem } from '../../../integration-tests/tests/utils/constants/wizard';
+import { testName } from '../../support';
+import { ProvisionSource } from '../../utils/const/provisionSource';
+import { row } from '../../views/selector';
+import {
+  wizardNavLink,
+  vmName,
+  templateProvider,
+  osDropdown,
+  imageSourceDropdown,
+  selectMenu,
+  sourceRegistry,
+  next,
+  customizeBtn,
+  cancelBtn,
+  templateTitle,
+} from '../../views/selector-wizard';
+
+const verifyWinGuestTools = () => {
+  cy.get(wizardNavLink)
+    .contains('Storage')
+    .should('not.be.disabled')
+    .click();
+  cy.get(row)
+    .contains('windows-guest-tools')
+    .should('exist');
+  cy.get(wizardNavLink)
+    .contains('General')
+    .click();
+  cy.get('#mount-windows-guest-tools-checkbox').uncheck();
+  cy.get(wizardNavLink)
+    .contains('Storage')
+    .click();
+  cy.get(row)
+    .contains('windows-guest-tools')
+    .should('not.exist');
+  cy.get(cancelBtn).click();
```

Comment:
Here it only exit the customize wizard, it's on the template select wizard.

Reply:
ok, moving to '/' before exiting